### DISCUSSION TITLE
C88D24A7-67FB-44F1-B7F3-350EC1D2CD07

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#40] [General]
# C88D24A7-67FB-44F1-B7F3-350EC1D2CD07

This addon adds, as an alternative surface, a topographic map for the planet Venus.

https://celestia.mobi/addon?item=C88D24A7-67FB-44F1-B7F3-350EC1D2CD07&lang=en

**Comment by SysQuirrel** [comment]
Not showing any topographic features on celestia mobile app

**Reply by DaveBowman2001** [reply]
Look for the Tab named `Alternate Surfaces` on its information page
